aiohttp/resolver.py

```diff
@@ -48,24 +51,24 @@ async def resolve(
                     # This is essential for link-local IPv6 addresses.
                     # LL IPv6 is a VERY rare case. Strictly speaking, we should use
                     # getnameinfo() unconditionally, but performance makes sense.
-                    host, _port = socket.getnameinfo(
-                        address, socket.NI_NUMERICHOST | socket.NI_NUMERICSERV
+                    resolved_host, _port = await self._loop.getnameinfo(
```

Comment:
This has apparently never worked on python less than 3.9

Reply:
Yeah, it's the same problem we have in zeroconf where scope ID isn't supported on an IP address until 3.9.

Sisters never been in any test coverage here, the problem has gone unnoticed since it's very rare. This is even used anyway nobody seems to have complained

Reply:
Huh, oh well, we'll drop support for 3.8 in a few months anyway.

Reply:
I'm going to mark the test as skip on 3.8 as I don't think there's any point in trying to fix it as it simply isn't going to work

I doubt anyone's using it on 3.8 who would even need this... I guess they really can't anyways